### PR TITLE
[PERFORMANCE] [master] Layer normalization code from Marian for CPU

### PR DESCRIPTION
**Pull request by kpuatamazon**
## Description ##
This is the master version of #19601.  There isn't much different in LayerNormalization implementation between v1.x and master.  

### Essentials ###
- [x] PR's title starts with a category (e.g. [BUGFIX], [MODEL], [TUTORIAL], [FEATURE], [DOC], etc)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage; already covered by unittest for layer normalization.
- [x] Code is well-documented

### Changes ###
- [x] Copy Marian optimized CPU LayerNorm implementation and adapt to MXNet.
- [x] Refactor dispatch of optimized versions using bool return value.
- [x] Remove MKL call

## Comments ##
See #19601 for benchmarks.

### CODE REVIEW

**Comment by kpuatamazon** [comment]
@mxnet-bot run ci [all]

Sigh everything is broken on some python HTTP thing.
```
[2020-11-30T20:24:22.242Z] Step 11/14 : RUN /work/docker_filepermissions.sh
[2020-11-30T20:25:18.379Z] [8924] Failed to execute script docker-compose
[2020-11-30T20:25:18.379Z] Traceback (most recent call last):
[2020-11-30T20:25:18.379Z]   File "http/client.py", line 554, in _get_chunk_left
[2020-11-30T20:25:18.379Z]   File "http/client.py", line 521, in _read_next_chunk_size
[2020-11-30T20:25:18.379Z] ValueError: invalid literal for int() with base 16: b''
[2020-11-30T20:25:18.379Z] 
[2020-11-30T20:25:18.379Z] During handling of the above exception, another exception occurred:
[2020-11-30T20:25:18.379Z] 
[2020-11-30T20:25:18.379Z] Traceback (most recent call last):
[2020-11-30T20:25:18.379Z]   File "http/client.py", line 586, in _readinto_chunked
[2020-11-30T20:25:18.379Z]   File "http/client.py", line 556, in _get_chunk_left
[2020-11-30T20:25:18.379Z] http.client.IncompleteRead: IncompleteRead(0 bytes read)
[2020-11-30T20:25:18.379Z] 
[2020-11-30T20:25:18.380Z] During handling of the above exception, another exception occurred:
[2020-11-30T20:25:18.380Z] 
[2020-11-30T20:25:18.380Z] Traceback (most recent call last):
[2020-11-30T20:25:18.380Z]   File "site-packages/urllib3/response.py", line 425, in _error_catcher
[2020-11-30T20:25:18.380Z]   File "site-packages/urllib3/response.py", line 507, in read
[2020-11-30T20:25:18.380Z]   File "http/client.py", line 457, in read
[2020-11-30T20:25:18.380Z]   File "http/client.py", line 491, in readinto
[2020-11-30T20:25:18.380Z]   File "http/client.py", line 602, in _readinto_chunked
[2020-11-30T20:25:18.380Z] http.client.IncompleteRead: IncompleteRead(0 bytes read)
[2020-11-30T20:25:18.380Z] 
[2020-11-30T20:25:18.380Z] During handling of the above exception, another exception occurred:
[2020-11-30T20:25:18.380Z] 
[2020-11-30T20:25:18.380Z] Traceback (most recent call last):
[2020-11-30T20:25:18.380Z]   File "bin/docker-compose", line 6, in <module>
[2020-11-30T20:25:18.380Z]   File "compose/cli/main.py", line 72, in main
[2020-11-30T20:25:18.380Z]   File "compose/cli/main.py", line 128, in perform_command
[2020-11-30T20:25:18.380Z]   File "compose/cli/main.py", line 303, in build
[2020-11-30T20:25:18.380Z]   File "compose/project.py", line 403, in build
[2020-11-30T20:25:18.380Z]   File "compose/project.py", line 385, in build_service
[2020-11-30T20:25:18.380Z]   File "compose/service.py", line 1110, in build
[2020-11-30T20:25:18.380Z]   File "compose/progress_stream.py", line 25, in stream_output
[2020-11-30T20:25:18.380Z]   File "compose/utils.py", line 61, in split_buffer
[2020-11-30T20:25:18.380Z]   File "compose/utils.py", line 37, in stream_as_text
[2020-11-30T20:25:18.380Z]   File "site-packages/docker/api/client.py", line 345, in _stream_helper
[2020-11-30T20:25:18.380Z]   File "site-packages/urllib3/response.py", line 529, in read
[2020-11-30T20:25:18.380Z]   File "contextlib.py", line 130, in __exit__
[2020-11-30T20:25:18.380Z]   File "site-packages/urllib3/response.py", line 443, in _error_catcher
[2020-11-30T20:25:18.380Z] urllib3.exceptions.ProtocolError: ('Connection broken: IncompleteRead(0 bytes read)', IncompleteRead(0 bytes read))
[2020-11-30T20:25:18.380Z] 2020-11-30 20:25:12,579 - root - WARNING - Exception: Command '['docker-compose', '-f', 'docker/docker-compose.yml', 'build', '--build-arg', 'USER_ID=1001', '--build-arg', 'GROUP_ID=1001', 'ubuntu_cpu']' returned non-zero exit status 255., Retrying in 1 seconds...
[2020-11-30T20:25:18.380Z] 2020-11-30 20:25:13,581 - root - INFO - Running command: 'docker-compose -f docker/docker-compose.yml build --build-arg USER_ID=1001 --build-arg GROUP_ID=1001 ubuntu_cpu'
[2020-11-30T20:25:18.380Z] Couldn't connect to Docker daemon at http+docker://localhost - is it running?
[2020-11-30T20:25:18.380Z] 
[2020-11-30T20:25:18.380Z] If it's at a non-standard location, specify the URL with the DOCKER_HOST environment variable.
[2020-11-30T20:25:18.380Z] 2020-11-30 20:25:14,126 - root - WARNING - Exception: Command '['docker-compose', '-f', 'docker/docker-compose.yml', 'build', '--build-arg', 'USER_ID=1001', '--build-arg', 'GROUP_ID=1001', 'ubuntu_cpu']' returned non-zero exit status 1., Retrying in 2 seconds...
[2020-11-30T20:25:18.380Z] 2020-11-30 20:25:16,129 - root - INFO - Running command: 'docker-compose -f docker/docker-compose.yml build --build-arg USER_ID=1001 --build-arg GROUP_ID=1001 ubuntu_cpu'
[2020-11-30T20:25:18.380Z] Couldn't connect to Docker daemon at http+docker://localhost - is it running?
[2020-11-30T20:25:18.380Z] 
[2020-11-30T20:25:18.380Z] If it's at a non-standard location, specify the URL with the DOCKER_HOST environment variable.
[2020-11-30T20:25:18.634Z] Traceback (most recent call last):
[2020-11-30T20:25:18.634Z]   File "ci/build.py", line 373, in <module>
[2020-11-30T20:25:18.634Z]     sys.exit(main())
[2020-11-30T20:25:18.634Z]   File "ci/build.py", line 309, in main
[2020-11-30T20:25:18.634Z]     no_cache=args.no_cache, cache_intermediate=args.cache_intermediate)
[2020-11-30T20:25:18.634Z]   File "ci/build.py", line 81, in build_docker
[2020-11-30T20:25:18.634Z]     run_cmd(env=env)
[2020-11-30T20:25:18.634Z]   File "/home/jenkins_slave/workspace/build-cpu-clang100/ci/util.py", line 84, in f_retry
[2020-11-30T20:25:18.634Z]     return f(*args, **kwargs)
[2020-11-30T20:25:18.634Z]   File "ci/build.py", line 79, in run_cmd
[2020-11-30T20:25:18.634Z]     check_call(cmd, env=env)
[2020-11-30T20:25:18.634Z]   File "/usr/lib/python3.6/subprocess.py", line 311, in check_call
[2020-11-30T20:25:18.634Z]     raise CalledProcessError(retcode, cmd)
[2020-11-30T20:25:18.634Z] subprocess.CalledProcessError: Command '['docker-compose', '-f', 'docker/docker-compose.yml', 'build', '--build-arg', 'USER_ID=1001', '--build-arg', 'GROUP_ID=1001', 'ubuntu_cpu']' returned non-zero exit status 1.
script returned exit code 1
```

**Comment by kpuatamazon** [comment]
@mxnet-bot run ci [unix-cpu, website, windows-cpu, windows-gpu]

Playing more CI docker daemon lottery.

**Comment by kpuatamazon** [comment]
Failed test on edge appears to be #19630.

**Comment by kpuatamazon** [comment]
Passing all tests except for `edge` which is broken per #19630..

**Comment by samskalicky** [comment]
> Passing all tests except for `edge` which is broken per #19630..

Retriggered after fix was completed, looks like its passing now.

**Comment by kpuatamazon** [comment]
@mxnet-bot run ci [unix-gpu]

Out of memory lottery ```[2020-12-21T12:32:00.010Z] c++: fatal error: Killed signal terminated program cc1plus```

**Comment by kpuatamazon** [comment]
@mxnet-bot run ci [centos-cpu]

Don't see how `tests/python/unittest/test_sparse_operator.py::test_sparse_unary_with_numerics` is related.  
```
[2020-12-21T12:56:11.413Z] Fatal Python error: Aborted
[2020-12-21T12:56:11.413Z] 
[2020-12-21T12:56:11.413Z] Thread 0x00007f1190ea2700 (most recent call first):
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/execnet/gateway_base.py", line 400 in read
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/execnet/gateway_base.py", line 432 in from_io
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/execnet/gateway_base.py", line 967 in _thread_receiver
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/execnet/gateway_base.py", line 220 in run
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/execnet/gateway_base.py", line 285 in _perform_spawn
[2020-12-21T12:56:11.413Z] 
[2020-12-21T12:56:11.413Z] Current thread 0x00007f1192b76740 (most recent call first):
[2020-12-21T12:56:11.413Z]   File "/work/mxnet/python/mxnet/autograd.py", line 269 in backward
[2020-12-21T12:56:11.413Z]   File "/work/mxnet/python/mxnet/executor.py", line 213 in backward
[2020-12-21T12:56:11.413Z]   File "/work/mxnet/python/mxnet/test_utils.py", line 1394 in check_symbolic_backward
[2020-12-21T12:56:11.413Z]   File "/work/mxnet/tests/python/unittest/test_sparse_operator.py", line 1502 in check_sparse_simple
[2020-12-21T12:56:11.413Z]   File "/work/mxnet/tests/python/unittest/test_sparse_operator.py", line 1518 in check_sparse_function
[2020-12-21T12:56:11.413Z]   File "/work/mxnet/tests/python/unittest/test_sparse_operator.py", line 1523 in test_sparse_unary_with_numerics
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/_pytest/python.py", line 184 in pytest_pyfunc_call
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/pluggy/callers.py", line 187 in _multicall
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/pluggy/manager.py", line 87 in <lambda>
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/pluggy/manager.py", line 93 in _hookexec
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/pluggy/hooks.py", line 286 in __call__
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/_pytest/python.py", line 1627 in runtest
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/_pytest/runner.py", line 163 in pytest_runtest_call
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/pluggy/callers.py", line 187 in _multicall
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/pluggy/manager.py", line 87 in <lambda>
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/pluggy/manager.py", line 93 in _hookexec
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/pluggy/hooks.py", line 286 in __call__
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/_pytest/runner.py", line 256 in <lambda>
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/_pytest/runner.py", line 310 in from_call
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/_pytest/runner.py", line 256 in call_runtest_hook
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/flaky/flaky_pytest_plugin.py", line 138 in call_and_report
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/_pytest/runner.py", line 127 in runtestprotocol
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/_pytest/runner.py", line 110 in pytest_runtest_protocol
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/flaky/flaky_pytest_plugin.py", line 94 in pytest_runtest_protocol
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/pluggy/callers.py", line 187 in _multicall
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/pluggy/manager.py", line 87 in <lambda>
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/pluggy/manager.py", line 93 in _hookexec
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/pluggy/hooks.py", line 286 in __call__
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/xdist/remote.py", line 87 in run_one_test
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/xdist/remote.py", line 70 in pytest_runtestloop
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/pluggy/callers.py", line 187 in _multicall
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/pluggy/manager.py", line 87 in <lambda>
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/pluggy/manager.py", line 93 in _hookexec
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/pluggy/hooks.py", line 286 in __call__
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/_pytest/main.py", line 313 in _main
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/_pytest/main.py", line 257 in wrap_session
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/_pytest/main.py", line 306 in pytest_cmdline_main
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/pluggy/callers.py", line 187 in _multicall
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/pluggy/manager.py", line 87 in <lambda>
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/pluggy/manager.py", line 93 in _hookexec
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/pluggy/hooks.py", line 286 in __call__
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/xdist/remote.py", line 237 in <module>
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/execnet/gateway_base.py", line 1084 in executetask
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/execnet/gateway_base.py", line 220 in run
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/execnet/gateway_base.py", line 285 in _perform_spawn
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/execnet/gateway_base.py", line 267 in integrate_as_primary_thread
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/execnet/gateway_base.py", line 1060 in serve
[2020-12-21T12:56:11.413Z]   File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/execnet/gateway_base.py", line 1554 in serve
[2020-12-21T12:56:11.413Z]   File "<string>", line 8 in <module>
[2020-12-21T12:56:11.413Z]   File "<string>", line 1 in <module>
[2020-12-21T12:56:11.667Z] tests/python/unittest/test_sparse_operator.py::test_sparse_unary_with_numerics 
[2020-12-21T12:56:11.667Z] [gw1] node down: Not properly terminated
[2020-12-21T12:56:16.898Z] [gw1] [ 93%] FAILED tests/python/unittest/test_sparse_operator.py::test_sparse_unary_with_numerics 
[2020-12-21T12:56:16.898Z] 
[2020-12-21T12:56:16.898Z] replacing crashed worker gw1
[2020-12-21T12:56:16.898Z] 
```

**Comment by kpuatamazon** [comment]
@mxnet-bot run ci [unix-cpu]

Memory roulette.  
```
[2020-12-21T12:39:34.723Z] c++: fatal error: Killed signal terminated program cc1plus
```

**Comment by kpuatamazon** [comment]
GPU is being flaky and CPU keeps running out of RAM @mxnet-bot run ci [unix-cpu, unix-gpu]

```
[2020-12-21T14:33:56.005Z] tests/python/gpu/test_operator_gpu.py::test_countsketch FAILED           [ 19%]

[2020-12-21T14:33:56.564Z] tests/python/gpu/test_operator_gpu.py::test_multi_sum_sq FAILED          [ 20%]

[2020-12-21T14:33:57.486Z] tests/python/gpu/test_operator_gpu.py::test_fast_lars FAILED             [ 20%]

[2020-12-21T14:33:57.740Z] tests/python/gpu/test_operator_gpu.py::test_batchnorm_with_type FAILED   [ 21%]

[2020-12-21T14:33:58.297Z] tests/python/gpu/test_operator_gpu.py::test_batchnorm_versions FAILED    [ 21%]

[2020-12-21T14:33:58.297Z] tests/python/gpu/test_operator_gpu.py::test_convolution_with_type FAILED [ 22%]

[2020-12-21T14:33:58.297Z] tests/python/gpu/test_operator_gpu.py::test_convolution_options SKIPPED  [ 22%]

[2020-12-21T14:33:58.854Z] tests/python/gpu/test_operator_gpu.py::test_conv_deconv_guards FAILED    [ 23%]

[2020-12-21T14:33:59.108Z] tests/python/gpu/test_operator_gpu.py::test_convolution_large_c FAILED   [ 23%]

[2020-12-21T14:33:59.362Z] tests/python/gpu/test_operator_gpu.py::test_deconvolution_large_c FAILED [ 24%]

[2020-12-21T14:33:59.362Z] tests/python/gpu/test_operator_gpu.py::test_convolution_versions FAILED  [ 24%]

[2020-12-21T14:33:59.615Z] tests/python/gpu/test_operator_gpu.py::test_pooling_nhwc_with_convention FAILED [ 25%]

[2020-12-21T14:33:59.869Z] tests/python/gpu/test_operator_gpu.py::test_pooling_with_type FAILED     [ 25%]

[2020-12-21T14:34:00.123Z] tests/python/gpu/test_operator_gpu.py::test_deconvolution_with_type FAILED [ 26%]

[2020-12-21T14:34:00.376Z] tests/python/gpu/test_operator_gpu.py::test_deconvolution_options FAILED [ 26%]

[2020-12-21T14:34:00.630Z] tests/python/gpu/test_operator_gpu.py::test_pooling_versions FAILED      [ 27%]

[2020-12-21T14:34:00.883Z] tests/python/gpu/test_operator_gpu.py::test_flatten_slice_after_conv FAILED [ 27%]

[2020-12-21T14:34:01.137Z] tests/python/gpu/test_operator_gpu.py::test_global_pooling FAILED        [ 28%]

[2020-12-21T14:34:01.391Z] tests/python/gpu/test_operator_gpu.py::test_psroipooling_with_type FAILED [ 28%]

[2020-12-21T14:34:01.644Z] tests/python/gpu/test_operator_gpu.py::test_deformable_psroipooling_with_type FAILED [ 29%]

[2020-12-21T14:34:01.898Z] tests/python/gpu/test_operator_gpu.py::test_deformable_convolution_with_type FAILED [ 29%]

[2020-12-21T14:34:02.819Z] tests/python/gpu/test_operator_gpu.py::test_sequence_reverse FAILED      [ 30%]

[2020-12-21T14:34:02.820Z] tests/python/gpu/test_operator_gpu.py::test_autograd_save_memory FAILED  [ 30%]

[2020-12-21T14:34:03.073Z] tests/python/gpu/test_operator_gpu.py::test_cuda_rtc FAILED              [ 31%]

[2020-12-21T14:34:03.327Z] tests/python/gpu/test_operator_gpu.py::test_cross_device_autograd FAILED [ 31%]

[2020-12-21T14:34:03.581Z] tests/python/gpu/test_operator_gpu.py::test_multi_proposal_op FAILED     [ 32%]
```

**Comment by kpuatamazon** [comment]
@mxnet-bot run ci [unix-cpu]

Memory gambling is annoying.

**Comment by kpuatamazon** [comment]
@mxnet-bot run ci [unix-cpu]

Still just running out of RAM compiling numpy kernels.  https://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/mxnet-validation%2Funix-cpu/detail/PR-19602/19/pipeline/

File: src/operator/nn/layer_norm.cc [hunk 1]
```diff
@@ -68,23 +99,127 @@ static bool LayerNormShape(const nnvm::NodeAttrs& attrs,
...
+ * mean is instances: means of each problem
+ * std is instances: standard deviation of each problem
+ *
+ */
+template <typename Data, typename Accum = typename
+            /* By default accumulate in float32 for float16.  Otherwise use same type. */
+            std::conditional<std::is_same<mshadow::half::half_t, Data>::value,
+                             float,
+                             Data>::type>
+void LayerNormCPUKernel(size_t width,
```
Comment (sxjscience) [inline src/operator/nn/layer_norm.cc:127]: I would recommend to change the name to `LayerNormContiguousCPUKernel` or `LayerNormLastAxisCPUKernel`

**Comment by sxjscience** [comment]
One naming issue. Looks good to me.

**Review by sxjscience — APPROVED** [review]
Minor issue (can be addressed later actually).

**Comment by fhieber** [comment]
What are the next steps for this PR? Is this ready to be merged?

**Comment by sxjscience** [comment]
@fhieber I've just merged. Feel free to try it out.

**Pull request merged**